internal/testutil/acc/shared_resource.go

```diff
@@ -89,10 +90,23 @@ func ProjectIDExecutionWithCluster(tb testing.TB, totalNodeCount int) (projectID
 // This function relies on using an execution project and returns its id.
 // When `MONGODB_ATLAS_CLUSTER_NAME` and `MONGODB_ATLAS_PROJECT_ID` are defined it will be used instead of creating resources. This is useful for local execution but not intended for CI executions.
 func ClusterNameExecution(tb testing.TB, populateSampleData bool) (projectID, clusterName string) {
+	tb.Helper()
+	return ClusterNameExecutionWithRegion(tb, "", populateSampleData)
```

Comment:
```suggestion
	return ClusterNameExecutionWithRegion(tb, constant.UsWest2, populateSampleData)
```
Will help with readability, makes it clear what region ClusterNameExecution uses without going into implementation.